### PR TITLE
🚀feat(tokens): add typescript in borders components

### PR DESCRIPTION
**Pull request by matheushdsbr**
## Description 📄

Add typescript script in border component

### Notes:
I'm keeping the tokens `index.js` file because when I change it to `index.ts` the '@gympass/yoga-tokens' module is not being found in `package/yoga/src/Theme/Provider/Provider.jsx`. 

On the code nothing happens, the error just appear when i execute the tests and when I try to execute the project with `yarn dev:web` i get another error, a generic error in `@gympass/yoga-doc`. I'm having trouble solving this case.

## Platforms 📲

- [x] Web
- [ ] Mobile

## Type of change 🔍

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [ ] Unit Test
- [ ] Snapshot Test

## Checklist: 🔍

- [x] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [ ] Layout matches design prototype: [FIGMA](https://figma.com/file/YOUR_LINK)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/tokens/src/global/font-sizes.ts [hunk 1]
```diff
@@ -25,7 +25,20 @@
  * @type {FontSize}
  * @default
  */
-const fontSizes = [10, 12, 14, 16, 20, 24, 32, 40, 48];
+
+interface fontSizesProps extends Array<number> {
```
Comment (MicaelRodrigues) [inline packages/tokens/src/global/font-sizes.ts:29]:
```suggestion
interface FontSizesProps extends number[] {
```

File: packages/tokens/src/global/font-weights.ts [hunk 2]
```diff
@@ -21,7 +21,16 @@
  * @type {FontWeight}
  * @default
  */
-const fontWeights = [300, 400, 500, 700, 900];
+
+interface fontWeightsProps extends Array<number> {
```
Comment (MicaelRodrigues) [inline packages/tokens/src/global/font-weights.ts:25]:
```suggestion
interface FontWeightsProps extends number[] {
```

File: packages/tokens/src/global/borders.ts [hunk 3]
```diff
@@ -19,8 +19,16 @@
...
+  zero?: number;
+  small?: number;
+  medium?: number;
+}
+
+const border: BorderProps = [0, 1, 2];
 
 [border.zero, border.small, border.medium] = border;
 
+
```
Comment (MicaelRodrigues) [inline packages/tokens/src/global/borders.ts:33]: You don't this extra line.

File: packages/tokens/src/global/line-heights.ts [hunk 4]
```diff
@@ -25,7 +25,20 @@
  * @type {LineHeight}
  * @default
  */
-const lineHeight = [12, 16, 20, 24, 28, 32, 40, 48, 56];
+
+interface lineHeightProps extends Array<number> {
```
Comment (MicaelRodrigues) [inline packages/tokens/src/global/line-heights.ts:29]:
```suggestion
interface LineHeightProps extends number[] {
```

File: packages/tokens/src/global/radii.ts [hunk 5]
```diff
@@ -21,7 +21,16 @@
  * @type {Radii}
  * @default
  */
-const radii = [0, 4, 8, 16, 9999];
+
+interface radiiProps extends Array<number> {
```
Comment (MicaelRodrigues) [inline packages/tokens/src/global/radii.ts:25]:
```suggestion
interface RadiiProps extends number[] {
```

File: packages/tokens/src/global/spacing.ts [hunk 6]
```diff
@@ -28,7 +28,23 @@
  * @type {Spacing}
  * @default
  */
-const spacing = [0, 4, 8, 12, 16, 20, 24, 32, 40, 56, 72, 80];
+
+ interface spacingProps extends Array<number> {
```
Comment (MicaelRodrigues) [inline packages/tokens/src/global/spacing.ts:32]:
```suggestion
 interface SpacingProps extends number[] {
```

File: packages/tokens/src/global/elevations.android.ts [hunk 7]
```diff
@@ -20,7 +20,14 @@
  * @type Elevation
  * @default
  */
-const elevations = [0, 4, 8, 12];
+ interface elevationsProps extends Array<number> {
```
Comment (MicaelRodrigues) [inline packages/tokens/src/global/elevations.android.ts:23]: Also, remove space before `interface`

**Review by guilhermecardoso-gympass — APPROVED** [review]

**Pull request merged**